client/.eslintrc.json

```diff
@@ -16,8 +16,7 @@
         "createDefaultProgram": true
       },
       "extends": [
-        "plugin:@angular-eslint/ng-cli-compat",
-        "plugin:@angular-eslint/ng-cli-compat--formatting-add-on",
+        "plugin:@angular-eslint/recommended",
```

Comment:
These changes were necessary to get `ng lint` to work. Presumably moving from v15 to v16 changes how the `@angular-eslint` plugin works.